src/Symfony/Component/Translation/Loader/CsvFileLoader.php

```diff
@@ -38,7 +38,7 @@ protected function loadResource(string $resource)
         }
 
         $file->setFlags(\SplFileObject::READ_CSV | \SplFileObject::SKIP_EMPTY);
-        $file->setCsvControl($this->delimiter, $this->enclosure, $this->escape);
+        $file->setCsvControl($this->delimiter, $this->enclosure, '' === $this->escape && \PHP_VERSION_ID < 70400 ? "\\" : $this->escape);
```

Comment:
```suggestion
        $file->setCsvControl($this->delimiter, $this->enclosure, '' === $this->escape && \PHP_VERSION_ID < 70400 ? '\\' : $this->escape);
```

Reply:
Mislead by the SPL stub :facepalm: Fixed where applicable